todo-app/src/App.js

```diff
@@ -1,13 +1,33 @@
 import React, { Component } from 'react';
+import Task from './Task';
+
+const todo_arr = [
+  {
+	 text:"Get out of bed" ,
+	 date:"Wed Sep 13 2017"
+  },
+  {
+	  text:"Brush teeth",
+	  date:"Thu Sep 14 2017"
+  },
+  {
+	  text:"Eat breakfast",
+	  date:"Fri Sep 15 2017"
+  }
+];
 
 class App extends Component {
+	
   render() {
     return (
       <div>
-        Hello World!
+        <h1>Todo list</h1>
+		<ul className='task-list'>
+			{todo_arr.map(item => <Task task={item} />)}
```

Comment:
You will notice if you open the browser console that React is warning you about keys. This is because you're iterating over your `todo_arr` with `map`, but not giving your returned top-level JSX keys.

You can check out [this section](https://reactjs.org/docs/lists-and-keys.html) of the React docs for more info. You need to give your `Task` components a unique key so React can tell when they are changed or removed.